src/validator.js

```diff
@@ -75,12 +75,23 @@ exports.validate = function(xmlData, options) {
           i--;
         }
         if (!validateTagName(tagName, regxTagName)) {
-          return {err: {code: 'InvalidTag', msg: 'Tag ' + tagName + ' is an invalid name.'}};
+          return {
```

Comment:
Though it is not related to your PR, can you please add this code?

```js
 if (!validateTagName(tagName, regxTagName)) {
          let msg;
          if(tagName.trim().length === 0) {
            msg = "There is an unnecessary space between tag name and backward slash '</ ..' ";
          }else{
            msg = `Tag ${tagName} is an invalid name.`;
          }
          return {
            err: {
              code: 'InvalidTag', msg: msg,
              line: getLineNumberForPosition(xmlData, i),
            },
          };
        }
```

This will elaborate the error when there is extra space between tagname and closing slash.

Reply:
Hi Amit,

same as below - this is unrelated and need additional tests and so forth.
We should probably create an issue and then merge that one separately.

Best regards
Norbert